src/app/components/cron-gen/cron-gen.jsx

```diff
@@ -169,14 +187,21 @@ function ChplCronGen(props) {
           <Typography variant="subtitle2">Schedule:</Typography>
           { cron && cronstrue.toString(cron, { verbose: true, dayOfWeekStartIndexZero: false }) }
         </div>
-        <Divider />
-        <Button disabled={dayType === 'daily'} onClick={() => setDayType('daily')}>Daily</Button>
-        <Button disabled={dayType === 'dayOfMonth'} onClick={() => setDayType('dayOfMonth')}>Day of Month</Button>
-        <Button disabled={dayType === 'nthWeekday'} onClick={() => setDayType('nthWeekday')}>Nth Weekday</Button>
-        <div className={classes.datetimeLayout}>
+        <AppBar elevation={1} position="static" color="transparent">
+          <Tabs
+            indicatorColor="primary"
+            textColor="primary"
+            variant="fullWidth"
+          >
+            <Tab label="Daily" onClick={() => setDayType('daily')} />
+            <Tab label="Day of Month" onClick={() => setDayType('dayOfMonth')} />
+            <Tab label="Nth Weekday" onClick={() => setDayType('nthWeekday')} />
+          </Tabs>
+        </AppBar>
+        <div>
```

Comment:
Is this `<div>` needed? It doesn't have a class, and all the divs that show up in here have one inside the various `dayType === 'something'` sections